src/Dlp/V2beta1/DlpServiceClient.php

```diff
@@ -0,0 +1,757 @@
+     *     $inspectConfig = new InspectConfig();
+     *     $storageConfig = new StorageConfig();
+     *     $outputConfig = new OutputStorageConfig();
+     *     $operationResponse = $dlpServiceClient->createInspectOperation($inspectConfig, $storageConfig, $outputConfig);
+     *     $operationResponse->pollUntilComplete();
+     *     if ($operationResponse->operationSucceeded()) {
+     *       $result = $operationResponse->getResult();
+     *       // doSomethingWith($result)
+     *     } else {
+     *       $error = $operationResponse->getError();
+     *       // handleError($error)
+     *     }
+     *
+     *     // OR start the operation, keep the operation name, and resume later
+     *     $operationResponse = $dlpServiceClient->createInspectOperation($inspectConfig, $storageConfig, $outputConfig);
+     *     $operationName = $operationResponse->getName();
+     *     // ... do other work
+     *     $newOperationResponse = $dlpServiceClient->resumeOperation($operationName, 'createInspectOperation');
+     *     while (!$newOperationResponse->isDone()) {
+     *         // ... do other work
+     *         $newOperationResponse->reload();
+     *     }
+     *     if ($newOperationResponse->operationSucceeded()) {
+     *       $result = $newOperationResponse->getResult();
+     *       // doSomethingWith($result)
+     *     } else {
+     *       $error = $newOperationResponse->getError();
+     *       // handleError($error)
+     *     }
+     * } finally {
+     *     $dlpServiceClient->close();
+     * }
+     * ```
+     *
+     * @param InspectConfig       $inspectConfig Configuration for the inspector.
+     * @param StorageConfig       $storageConfig Specification of the data set to process.
+     * @param OutputStorageConfig $outputConfig  Optional location to store findings. The bucket must already exist and
+     *                                           the Google APIs service account for DLP must have write permission to
+     *                                           write to the given bucket.
+     *                                           <p>Results are split over multiple csv files with each file name matching
+     *                                           the pattern "[operation_id]_[count].csv", for example
+     *                                           `3094877188788974909_1.csv`. The `operation_id` matches the
+     *                                           identifier for the Operation, and the `count` is a counter used for
+     *                                           tracking the number of files written. <p>The CSV file(s) contain the
+     *                                           following columns regardless of storage type scanned: <li>id <li>info_type
+     *                                           <li>likelihood <li>byte size of finding <li>quote <li>time_stamp<br/>
+     *                                           <p>For Cloud Storage the next columns are: <li>file_path
```

Comment:
I know technically it isn't required, but WDYT about closing tags for `<p>` and `<li>` elements? If we could go a step further, maybe we should just drop the tags altogether? The flow of this documentation is a little tricky to follow, reading from the source here.

For example, node comma separates the values: https://github.com/GoogleCloudPlatform/google-cloud-node/blob/master/packages/dlp/src/v2beta1/dlp_service_client.js#L304-L308